src/tvheadend/Settings.h

```diff
@@ -106,6 +107,7 @@ namespace tvheadend {
     : m_strHostname(DEFAULT_HOST),
       m_iPortHTSP(DEFAULT_HTTP_PORT),
       m_iPortHTTP(DEFAULT_HTSP_PORT),
+      m_buseHTTPS(DEFAULT_USE_HTTPS),
```

Comment:
Wrong member name here, should be `m_bUseHTTPS`

Reply:
I should really compile this before pushing... :'-)